daemon/pod/provision.go

```diff
@@ -211,7 +214,7 @@ func (p *XPod) ContainerStart(cid string) error {
 // Start() means start a STOPPED pod.
 func (p *XPod) Start() error {
 
-	if p.status == S_POD_STOPPED {
+	if p.IsStopped() {
```

Comment:
let's leave this status in later issue/pr. 

In principle, if a pod failed and could be restarted, it should be set back to stopped status. Only those meet fatal exception and could not start any more should stay in ERROR status. 

However, sometimes the daemon can not judge the situation accurately. And I think it might be better if we could allow a user to set back status from ERROR to STOPPED manually.